adm/templates/plugins/django/{{cookiecutter.name}}/python3_virtualenv_sources/.gitignore

```diff
@@ -0,0 +1,2 @@
+requirements3.txt
+src
```

Comment:
please put a test like {% if cookiecutter.type == "gunicorn3_sync" %} before these lines
if we selected python2, the file will be empty (with this test) and deleted as the python3_virtualenv_sources directory

Reply:
done